frontend/server/src/main/java/com/amazonaws/ml/mms/util/logging/QLogLayout.java

```diff
@@ -26,15 +26,19 @@ public String format(LoggingEvent event) {
             return null;
         }
         if (eventMessage instanceof Metric) {
+            String marketPlace = System.getenv("REALM");
             StringBuilder stringBuilder = new StringBuilder();
             Metric metric = (Metric) eventMessage;
             stringBuilder.append("HostName=").append(metric.getHostName());
             if (metric.getRequestId() != null) {
                 stringBuilder.append("\nRequestId=").append(metric.getRequestId());
             }
-            stringBuilder.append("\nTimeStamp=").append(metric.getTimestamp());
+            if (marketPlace != null) {
+                stringBuilder.append("\nMarketPlace=").append(marketPlace);
```

Comment:
We don't seem to be populating the Marketplace if "REALM" is not given, shouldn't we have it set to "NONE" as a value? Doesn't metrics expect a string in return? Also, is the spelling of Marketplace correct. The reference says it should be "Marketplace" (`p` is small not capital)

Reply:
By default if not provided it is set to NONE automatically and you are right it is Metricname